2018-04-16 17:54:11,420 2548-123145366478848 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-16 17:58:12,269 2548-123145370685440 INFO: A new user created, User({'status': u'normal', 'password': u'pbkdf2:sha256:50000$XYNpM8F4$744c5693e15efdb2b1fbd55029167bd7ebd5cbb9b206b17654a6dfba8df1e84e', 'name': u'1965258618', 'roles': [1], 'point': 0, 'email': u'1965258618@qq.com', '_id': ObjectId('5ad473b4a534ff09f4c8c308'), 'createTime': datetime.datetime(2018, 4, 16, 17, 58, 12, 24138)}) [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:138]
2018-04-16 17:58:12,270 2548-123145370685440 INFO: Try to send support email [Info] flask-boot.com: signup() on 2018/04/16 [DEV] to ['okosioc@gmail.com'] [in /Users/lgp/Desktop/python-workspace/demo/app/tools/notifier.py:34]
2018-04-16 18:24:41,131 2624-123145563320320 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-16 19:09:57,586 3372-123145344401408 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-16 19:42:23,567 3521-123145419612160 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-16 19:42:41,685 3521-123145419612160 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-16 19:42:44,552 3521-123145419612160 DEBUG: There are 0 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-16 19:42:46,065 3521-123145419612160 DEBUG: There are 0 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-16 19:43:54,391 3521-123145419612160 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-16 19:44:38,949 3521-123145419612160 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-16 19:44:42,470 3521-123145419612160 DEBUG: There are 1 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-16 19:44:43,837 3521-123145419612160 DEBUG: There are 1 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-16 19:46:39,886 3521-123145419612160 DEBUG: There are 2 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-16 19:46:51,129 3521-123145419612160 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-20 11:47:33,609 5217-123145417515008 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 12:07:34,414 5217-123145417515008 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 12:51:10,124 5217-123145417515008 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 13:29:31,126 5217-123145417515008 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 13:49:32,028 5217-123145417515008 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 14:29:35,920 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 14:49:36,487 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 15:09:37,166 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 15:29:38,154 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 15:49:38,984 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 16:09:39,955 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 16:29:40,918 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 16:49:40,972 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 18:32:53,208 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 19:08:45,243 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 20:43:25,239 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 21:35:13,244 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 21:57:26,242 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 22:24:04,243 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 22:51:05,241 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 23:11:06,187 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 23:31:06,853 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-20 23:51:07,240 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 00:14:04,212 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 00:41:05,220 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 01:07:25,213 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 01:37:26,210 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 01:57:26,224 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 02:17:26,881 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 02:39:48,215 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 03:01:07,358 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 03:27:04,493 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 03:51:06,480 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 04:11:07,463 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 04:37:02,502 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 05:01:30,301 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 05:21:30,384 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 05:47:01,470 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 06:17:01,474 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 06:41:05,474 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 07:04:04,470 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 07:34:04,310 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 08:07:30,507 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 08:40:23,406 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 09:01:03,473 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 09:21:04,286 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 09:50:00,440 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 10:16:02,484 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 10:39:59,445 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 11:01:02,427 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 11:21:02,505 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 11:41:03,195 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 23:06:17,545 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 23:30:59,196 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-21 23:51:00,182 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 00:21:05,251 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 00:41:06,254 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 01:01:26,249 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 01:21:26,828 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 01:41:50,259 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 02:11:26,261 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 02:31:26,808 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 02:51:50,265 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 05:09:43,516 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 05:32:05,513 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 06:01:07,577 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 06:23:17,568 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 06:53:06,543 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 07:21:26,552 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 07:41:27,101 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 08:01:55,580 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 08:21:56,455 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 08:41:57,292 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 09:01:58,103 5369-123145458327552 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 09:25:32,652 34839-123145494978560 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 09:49:31,402 34931-123145545564160 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 09:49:52,110 34931-123145545564160 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:03:44,319 34931-123145541357568 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 10:05:27,370 34931-123145545564160 INFO: A new user created, User({'status': u'normal', 'password': u'pbkdf2:sha256:50000$oysUmq8g$23734cb97b5d6d1e928f610043042f85470791f9c025cc969a93ddb123211a91', 'name': u'1965258618', 'roles': [1], 'point': 0, 'email': u'1965258618@qq.com', '_id': ObjectId('5adbede7ef803b88735bfe9e'), 'createTime': datetime.datetime(2018, 4, 22, 10, 5, 27, 88527)}) [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:131]
2018-04-22 10:05:27,373 34931-123145545564160 INFO: Try to send support email [Info] flask-boot.com: signup() on 2018/04/22 [DEV] to ['okosioc@gmail.com'] [in /Users/lgp/Desktop/python-workspace/demo/app/tools/notifier.py:34]
2018-04-22 10:05:27,383 34931-123145545564160 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:05:35,572 34931-123145545564160 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:05:40,497 34931-123145545564160 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:06:24,600 34993-123145424654336 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:06:29,503 34993-123145424654336 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:08:04,891 34993-123145424654336 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:08:19,435 34993-123145424654336 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:08:21,966 34993-123145424654336 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:08:28,044 34993-123145424654336 DEBUG: There are 2 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:08:32,740 34993-123145424654336 DEBUG: There are 0 tags for condition {}, with projection {'_id': True, 'name': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:08:33,722 34993-123145424654336 DEBUG: There are 0 posts for condition {}, with projection {'_id': True, 'createTime': True, 'tids': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:08:34,895 34993-123145424654336 DEBUG: There are 0 keywords for condition {}, with projection {'name': True, 'level': True, 'baiduIndex': True, 'parentId': True, 'owner': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:08:35,971 34993-123145424654336 DEBUG: There are 0 configs for condition {}, with projection {'_id': True, 'name': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:08:37,906 34993-123145424654336 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:08:38,688 34993-123145424654336 DEBUG: There are 2 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:08:43,791 34993-123145424654336 DEBUG: There are 2 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:28:20,329 34993-123145420447744 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 10:30:26,757 34993-123145424654336 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:30:32,044 34993-123145424654336 DEBUG: There are 2 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:38:52,479 34993-123145424654336 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 10:40:15,243 34993-123145424654336 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:40:17,343 34993-123145424654336 DEBUG: There are 3 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 10:48:20,894 34993-123145420447744 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 11:00:27,938 34993-123145424654336 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 11:26:50,334 34993-123145420447744 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 11:28:53,269 34993-123145424654336 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 11:29:45,781 34993-123145424654336 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 11:30:24,724 34993-123145424654336 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:00:00,517 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:00:12,588 35414-123145444896768 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 14:00:40,372 35414-123145444896768 INFO: A new user created, User({'status': u'normal', 'password': u'pbkdf2:sha256:50000$WTbbf7DF$701757a2634652b6e2264a03a87ab7aca19c08f34ba84c5126236856a7090a72', 'name': u'1965258618', 'roles': [1], 'point': 0, 'email': u'1965258618@qq.com', '_id': ObjectId('5adc2508ef803b8a56cf3d8b'), 'createTime': datetime.datetime(2018, 4, 22, 14, 0, 40, 145423)}) [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:131]
2018-04-22 14:00:40,373 35414-123145444896768 INFO: Try to send support email [Info] flask-boot.com: signup() on 2018/04/22 [DEV] to ['okosioc@gmail.com'] [in /Users/lgp/Desktop/python-workspace/demo/app/tools/notifier.py:34]
2018-04-22 14:00:40,381 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:00:52,368 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:01:53,774 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:01:59,947 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:02:04,343 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:02:15,625 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:06:08,980 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:06:34,595 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:07:31,276 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:09:14,338 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:09:47,008 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:09:57,448 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:10:30,764 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:15:03,879 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:15:11,203 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:19:55,514 35414-123145440690176 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 14:21:05,183 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:28:13,965 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:28:18,722 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:29:25,773 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:29:33,997 35414-123145444896768 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:29:36,338 35414-123145444896768 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 14:29:41,293 35414-123145444896768 DEBUG: There are 3 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 14:41:20,072 35477-123145580511232 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:42:05,959 35477-123145580511232 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 14:45:28,123 35499-123145529401344 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 15:22:44,225 35664-123145495556096 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 15:24:24,901 35682-123145552289792 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 15:44:18,836 35682-123145548083200 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 15:52:09,255 35682-123145552289792 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:01:33,570 35750-123145369944064 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:12:14,689 35862-123145481043968 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:17:42,651 35900-123145338687488 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:34:05,117 35964-123145324498944 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:35:49,384 35973-123145569726464 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:37:25,691 35997-123145322557440 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:38:12,884 36011-123145543856128 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:44:47,524 36031-123145455169536 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 16:55:09,085 36085-123145360691200 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 17:31:08,372 36085-123145356484608 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 17:51:08,811 36085-123145356484608 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 17:53:32,797 36145-123145386758144 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 17:55:54,542 36163-123145450332160 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:02:11,887 36202-123145329639424 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:02:58,139 36218-123145498480640 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:20:11,978 36241-123145349238784 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:20:57,471 36252-123145348780032 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:36:49,574 36305-123145510150144 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 18:36:51,360 36305-123145510150144 DEBUG: There are 4 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 18:37:36,875 36319-123145559650304 DEBUG: There are 1 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 18:37:37,901 36319-123145559650304 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:37:46,642 36319-123145559650304 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:47:33,234 36350-123145456455680 INFO: Found 4 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:47:42,972 36350-123145456455680 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 18:47:44,573 36350-123145456455680 DEBUG: There are 4 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 18:54:23,483 36374-123145413615616 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:54:33,980 36374-123145413615616 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 18:54:58,993 36374-123145413615616 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 18:55:02,312 36374-123145413615616 DEBUG: There are 2 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 18:59:36,914 36418-123145432457216 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:13:14,887 36447-123145394921472 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:13:20,020 36447-123145394921472 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:13:22,086 36447-123145394921472 DEBUG: There are 2 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:13:27,607 36447-123145394921472 INFO: Found 1 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:13:38,088 36447-123145394921472 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:13:50,582 36447-123145394921472 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:13:57,148 36447-123145394921472 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:13:58,491 36447-123145394921472 DEBUG: There are 3 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:19:18,332 36463-123145516179456 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:19:49,496 36463-123145516179456 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:19:51,757 36463-123145516179456 DEBUG: There are 3 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:21:01,786 36481-123145557856256 INFO: Found 3 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:23:26,169 36481-123145557856256 INFO: Found 4 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:23:37,260 36481-123145557856256 INFO: Found 5 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:28:51,171 36481-123145557856256 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:28:52,595 36481-123145557856256 DEBUG: There are 5 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:29:08,892 36481-123145557856256 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:29:50,766 36481-123145557856256 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:38:24,063 36542-123145373188096 INFO: Found 2 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:38:26,951 36542-123145373188096 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 19:45:51,884 36542-123145373188096 DEBUG: There are 1 users for condition {}, with projection {'_id': True, 'email': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:45:53,548 36542-123145373188096 DEBUG: There are 285 prices for condition {}, with projection {'date': True, '_id': True} [in /Users/lgp/Desktop/python-workspace/demo/app/views/crud.py:80]
2018-04-22 19:58:15,206 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 20:18:15,922 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 20:38:16,760 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 22:23:52,393 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 22:56:03,395 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 23:16:03,546 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 23:21:54,971 36542-123145373188096 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-22 23:36:04,250 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-22 23:56:04,405 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 00:16:04,664 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 00:36:04,719 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 01:01:05,472 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 01:21:27,478 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 01:41:28,000 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 02:08:15,481 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 02:31:26,483 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 02:51:27,491 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 03:12:05,484 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 03:35:05,478 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 04:07:51,920 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 04:37:33,775 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 05:01:07,764 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 05:26:07,702 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 05:46:08,709 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 07:17:42,790 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 07:44:06,491 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 08:52:20,500 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 08:59:26,568 36542-123145373188096 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 09:12:21,178 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 09:32:22,029 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 09:43:44,352 36542-123145373188096 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 09:52:22,634 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 10:12:23,246 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 10:32:23,373 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 10:52:23,903 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 11:12:24,075 36542-123145368981504 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 11:52:39,577 37391-123145438818304 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 12:16:34,203 37445-123145526624256 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 12:36:35,129 37445-123145526624256 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 12:56:35,987 37445-123145526624256 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 13:47:54,711 37445-123145526624256 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 14:07:55,078 37445-123145526624256 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 14:27:55,338 37445-123145526624256 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 14:47:56,312 37445-123145526624256 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 15:27:32,581 37737-123145471873024 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 15:47:32,637 37737-123145471873024 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 16:07:33,602 37737-123145471873024 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 17:02:55,318 38161-123145379622912 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:03:29,070 38161-123145379622912 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:05:33,281 38201-123145492623360 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:06:55,467 38218-123145455345664 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:07:04,788 38218-123145455345664 INFO: Found 286 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:07:50,549 38229-123145333481472 INFO: Found 287 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:09:41,332 38240-123145484681216 INFO: Found 290 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:12:00,392 38240-123145484681216 INFO: Found 293 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:12:49,564 38253-123145402826752 INFO: Found 296 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:23:04,663 38277-123145584467968 INFO: Found 299 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:28:05,284 38286-123145326223360 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:28:09,783 38286-123145326223360 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-23 17:44:53,372 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 19:41:38,862 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 20:59:21,860 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-23 22:21:51,319 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 00:41:48,144 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 01:07:04,952 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 03:15:47,124 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 03:36:08,161 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 04:06:28,094 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 04:35:33,125 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 05:05:30,127 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 05:37:02,090 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 06:00:18,142 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 09:03:35,129 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 09:23:35,310 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 09:43:36,200 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 09:48:09,637 38286-123145326223360 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 09:59:41,354 38286-123145326223360 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:01:06,072 38286-123145326223360 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:02:21,448 38286-123145326223360 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:03:26,597 38286-123145326223360 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:03:36,933 38286-123145322016768 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:28]
2018-04-24 10:06:34,753 38286-123145326223360 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:10:57,869 38802-123145574477824 INFO: Found 200 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:12:22,531 38802-123145574477824 INFO: Found 200 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:12:25,518 38802-123145574477824 INFO: Found 200 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:12:48,503 38802-123145574477824 INFO: Found 200 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:12:50,449 38802-123145574477824 INFO: Found 300 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:13:46,571 38802-123145574477824 INFO: Found 300 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:13:57,158 38802-123145574477824 INFO: Found 400 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:15:50,212 38802-123145574477824 INFO: Found 400 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:15:57,677 38802-123145574477824 INFO: Found 400 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:16:14,459 38802-123145574477824 INFO: Found 500 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:17:17,116 38802-123145574477824 INFO: Found 500 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:17:19,999 38802-123145574477824 INFO: Found 600 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:21:21,890 38802-123145574477824 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:22:16,038 38802-123145574477824 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:25:40,270 38852-123145466445824 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:25:51,633 38852-123145466445824 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:31:48,830 38852-123145466445824 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:31:53,351 38852-123145466445824 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:33:41,817 38879-123145373159424 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:33:44,068 38879-123145373159424 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:34:03,128 38879-123145373159424 INFO: Found 200 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:36:01,371 38890-123145420300288 INFO: Found 300 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:38:00,487 38913-123145384984576 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:38:16,846 38913-123145384984576 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:41:07,274 38948-123145434820608 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:41:16,570 38948-123145434820608 INFO: Found 200 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:43:17,190 38964-123145528123392 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:44:26,150 38974-123145530048512 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:44:27,875 38974-123145530048512 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 10:44:31,352 38974-123145530048512 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 11:25:11,265 39126-123145479323648 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 11:26:05,300 39126-123145475117056 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:26:05,301 39126-123145475117056 ERROR: Failed when updating the prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:26:52,477 39126-123145479323648 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 11:27:05,559 39126-123145475117056 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:27:05,560 39126-123145475117056 ERROR: Failed when updating the prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:27:06,231 39126-123145479323648 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 11:28:05,771 39126-123145475117056 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:28:05,772 39126-123145475117056 ERROR: Failed when updating the prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:29:18,157 39139-123145536524288 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:29:18,158 39139-123145536524288 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:30:18,371 39139-123145536524288 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:30:18,372 39139-123145536524288 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:32:02,041 39151-123145522499584 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:32:02,042 39151-123145522499584 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:41:43,955 39221-123145499729920 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:41:43,962 39221-123145499729920 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    blog.kline_huobi(app)
NameError: global name 'blog' is not defined
2018-04-24 11:42:44,190 39221-123145499729920 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:42:44,191 39221-123145499729920 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    blog.kline_huobi(app)
NameError: global name 'blog' is not defined
2018-04-24 11:43:44,401 39221-123145499729920 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:43:44,402 39221-123145499729920 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    blog.kline_huobi(app)
NameError: global name 'blog' is not defined
2018-04-24 11:44:44,659 39221-123145499729920 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:44:44,660 39221-123145499729920 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    blog.kline_huobi(app)
NameError: global name 'blog' is not defined
2018-04-24 11:45:43,864 39221-123145499729920 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:55]
2018-04-24 11:45:43,865 39221-123145499729920 ERROR: Failed when updating binance the prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:59]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 57, in binance_update_prices
    blog.kline_binance(app)
NameError: global name 'blog' is not defined
2018-04-24 11:45:44,886 39221-123145499729920 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:45:44,886 39221-123145499729920 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    blog.kline_huobi(app)
NameError: global name 'blog' is not defined
2018-04-24 11:46:18,950 39221-123145503936512 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:152]
2018-04-24 11:46:45,106 39221-123145499729920 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:46:45,112 39221-123145499729920 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    blog.kline_huobi(app)
NameError: global name 'blog' is not defined
2018-04-24 11:47:45,332 39221-123145499729920 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:47:45,333 39221-123145499729920 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    blog.kline_huobi(app)
NameError: global name 'blog' is not defined
2018-04-24 11:49:07,337 39239-123145318764544 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:49:07,337 39239-123145318764544 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:50:30,696 39259-123145474834432 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:50:30,696 39259-123145474834432 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:51:30,912 39259-123145474834432 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:51:30,918 39259-123145474834432 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:52:31,138 39259-123145474834432 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:52:31,139 39259-123145474834432 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi(app)
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:54:25,473 39280-123145497669632 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:54:25,473 39280-123145497669632 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi()
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:55:25,716 39280-123145497669632 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:55:25,722 39280-123145497669632 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    kline_huobi()
NameError: global name 'kline_huobi' is not defined
2018-04-24 11:58:27,012 39315-123145490440192 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:58:27,014 39315-123145490440192 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    app.views.blog.kline_huobi()
AttributeError: 'Flask' object has no attribute 'views'
2018-04-24 11:59:27,209 39315-123145490440192 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:45]
2018-04-24 11:59:27,210 39315-123145490440192 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:49]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 47, in huobi_update_prices
    app.views.blog.kline_huobi()
AttributeError: 'Flask' object has no attribute 'views'
2018-04-24 12:01:21,574 39337-123145467723776 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:46]
2018-04-24 12:01:21,574 39337-123145467723776 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:50]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 48, in huobi_update_prices
    app.views.blog.kline_huobi()
AttributeError: 'Flask' object has no attribute 'views'
2018-04-24 12:03:05,437 39353-123145542750208 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:46]
2018-04-24 12:03:05,438 39353-123145542750208 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:50]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 48, in huobi_update_prices
    kline_huobi()
NameError: global name 'kline_huobi' is not defined
2018-04-24 12:04:05,693 39353-123145542750208 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:46]
2018-04-24 12:04:05,694 39353-123145542750208 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:50]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 48, in huobi_update_prices
    kline_huobi()
NameError: global name 'kline_huobi' is not defined
2018-04-24 12:37:28,781 39502-123145518641152 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:37:29,442 39502-123145518641152 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 215, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:37:59,643 39502-123145518641152 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:38:00,285 39502-123145518641152 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 215, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:38:30,404 39502-123145518641152 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:38:33,840 39502-123145518641152 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 215, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:39:03,942 39502-123145518641152 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:39:04,489 39502-123145518641152 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 215, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:39:34,581 39517-123145561493504 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:39:40,984 39517-123145561493504 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:39:50,724 39517-123145557286912 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:39:51,536 39517-123145557286912 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 215, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:40:21,649 39517-123145557286912 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:40:22,365 39517-123145557286912 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 215, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:40:52,498 39517-123145557286912 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:40:53,188 39517-123145557286912 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 215, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:41:23,308 39517-123145557286912 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:41:23,630 39517-123145557286912 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 215, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:41:53,747 39517-123145557286912 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:42:21,360 39544-123145410609152 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:42:44,492 39544-123145406402560 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:42:45,022 39544-123145406402560 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:43:15,162 39544-123145406402560 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:43:15,743 39544-123145406402560 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:43:33,635 39544-123145410609152 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:43:45,855 39544-123145406402560 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:43:46,461 39544-123145406402560 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:43:51,664 39544-123145410609152 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:43:55,802 39544-123145410609152 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:44:07,912 39544-123145410609152 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:44:16,587 39544-123145406402560 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:44:17,181 39544-123145406402560 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:44:47,340 39544-123145406402560 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:44:47,647 39544-123145406402560 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:45:17,755 39544-123145406402560 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 12:45:18,415 39544-123145406402560 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 12:45:57,682 39571-123145333350400 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:46:55,607 39571-123145333350400 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:47:21,693 39571-123145333350400 INFO: Found 100 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:47:50,253 39571-123145333350400 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:48:00,512 39571-123145333350400 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 12:48:21,801 39571-123145333350400 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 13:02:26,320 39571-123145333350400 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 13:05:49,350 39571-123145329143808 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:31]
2018-04-24 13:25:50,131 39571-123145329143808 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:31]
2018-04-24 13:45:50,882 39571-123145329143808 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:31]
2018-04-24 14:05:51,892 39571-123145329143808 INFO: Scheduler update_view_times running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:31]
2018-04-24 14:23:01,614 39571-123145333350400 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 14:23:52,544 39571-123145333350400 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 14:24:40,307 39682-123145382805504 INFO: Found 0 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 14:24:49,324 39682-123145382805504 INFO: Found 285 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 14:24:57,021 39682-123145382805504 INFO: Found 385 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 14:25:04,826 39682-123145382805504 INFO: Found 385 prices [in /Users/lgp/Desktop/python-workspace/demo/app/views/public.py:158]
2018-04-24 14:30:54,465 39707-123145543581696 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 14:30:54,992 39707-123145543581696 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 14:31:25,144 39707-123145543581696 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 14:31:25,444 39707-123145543581696 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 14:31:55,548 39707-123145543581696 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 14:31:56,161 39707-123145543581696 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 14:32:26,272 39707-123145543581696 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 14:32:26,845 39707-123145543581696 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 14:32:56,996 39707-123145543581696 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
2018-04-24 14:32:57,574 39707-123145543581696 ERROR: Failed when updating the huobi prices [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:52]
Traceback (most recent call last):
  File "/Users/lgp/Desktop/python-workspace/demo/app/jobs.py", line 50, in huobi_update_prices
    kline_huobi(app)
  File "/Users/lgp/Desktop/python-workspace/demo/app/views/public.py", line 217, in kline_huobi
    return jsonify(success=True, message=_('Save the post successfully.'))
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/json.py", line 251, in jsonify
    if current_app.config['JSONIFY_PRETTYPRINT_REGULAR'] and not request.is_xhr:
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/werkzeug/local.py", line 306, in _get_current_object
    return self.__local()
  File "/Users/lgp/Desktop/python-workspace/venv/lib/python2.7/site-packages/flask/globals.py", line 51, in _find_app
    raise RuntimeError(_app_ctx_err_msg)
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
to interface with the current application object in a way.  To solve
this set up an application context with app.app_context().  See the
documentation for more information.
2018-04-24 14:33:32,773 39728-123145414512640 INFO: Scheduler update_prices running: Counter() [in /Users/lgp/Desktop/python-workspace/demo/app/jobs.py:48]
